Read-and-Save-document/Open-and-save-Word-document/Blazor/Client-side-application/Open-and-save-Word-document/Pages/DocIO.razor.bak

```diff
@@ -0,0 +1,37 @@
+﻿@page "/DocIO"
```

Comment:
Remove this unwanted file DocIO.razor.bak

Reply:
Addressed the requested changes